.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "query-protocol"]
 	path = query-protocol
 	url = http://github.com/apache/skywalking-query-protocol/
+    branch = master
```

Comment:
What is this?

Reply:
I haven't used submodules, ci errors, keep trying.

Reply:
> I haven't used submodules, ci errors, keep trying.

This should be reverted, it doesn't fix the CI, see my changes above